actions/get_dependencies/get_dependencies.sh

```diff
@@ -168,7 +188,7 @@ get_libint() {
     cd libint-2.6.0
     export CXX=`which g++`
     export CC=`which gcc`
-    ../cmake-3.16.3-Linux-x86_64/bin/cmake -H. -Bbuild -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_FLAGS="-std=c++17" -DBUILD_SHARED_LIBS=ON -DCPP_GITHUB_TOKEN=$CPP_GITHUB_TOKEN
+    ../cmake-3.16.3-Linux-x86_64/bin/cmake -H. -Bbuild -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_FLAGS="-std=c++17" -DBUILD_SHARED_LIBS=ON -DCMAIZE_GITHUB_TOKEN=$CMAIZE_GITHUB_TOKEN
```

Comment:
Why do you need to pass all these flags to libint's build?

Reply:
It was there before. I only changed cpp_github_token to cmaize_github_token. Ok, I can try to build libint without using these and see if it works.

Reply:
You're right. Not sure why we (probably me) allowed that to be committed in the first place (maybe it was supposed to get subsequently cleaned up?).

Reply:
No worries. I can clean up some of those. Hehe.